src/test/packages/podinfo/zarf.yaml

```diff
@@ -23,3 +23,47 @@ components:
                 name: podinfo
                 namespace: podinfo
                 condition: available
+
+  - name: podinfo-flavor
+    required: true
+    only:
+      flavor: two
```

Comment:
Pretty sure `two` here refers to the patch version in 6.6.2, could we make that more apparent in the flavor name? Something like patchVersion2 or something like that?

Reply:
updated the naming for flavor `two` and `three`